### ISSUE TITLE
Accessability Violation : The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic"(ibm-code-snippet)

### ISSUE THREAD

**Issue by GSSCDylan** [carbon-design-system/carbon-components-angular#2550]
## Detailed description

> Describe in detail the issue you're having.

The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic"

> Is this a feature request (new component, new icon), a bug, or a general issue?

 bug

> Is this issue related to a specific component?

ibm-code-snippet

> What did you expect to happen? What happened instead? What would you like to see changed?

 Need to remove aria-label in div tag of ibm-code-snippet
> What browser are you working in?

Firefox
> What version of the Carbon Design System are you using?

    "carbon-components": "^10.58.3",
    "carbon-components-angular": "^4.71.8",

https://angular.carbondesignsystem.com/?path=/story/components-code-snippet--basic

## Additional information

* Screenshots or code

<img width="1178" alt="aria" src="https://user-images.githubusercontent.com/73825731/236821607-a35e949b-82d6-4b79-986f-ff4ffb37c77d.png">
